docs/extending/adding_reports.md

```diff
@@ -49,7 +49,7 @@ class UnpublishedChangesReportView(PageReportView):
 The template used to render your report. For ``ReportView``, this defaults to ``"wagtailadmin/reports/base_report.html"``,
 which provides an empty report page layout; for ``PageReportView``, this defaults to
 ``"wagtailadmin/reports/base_page_report.html"`` which provides a listing based on the explorer views,
-displaying action buttons, as well as the title, time of the last update, status, and specific type of any pages.
+displaying action buttons, as well as the title, time of the last update, status, and specific type of pages.
```

Comment:
Not sure on this change. We are referring to a list of things and then an optional set of pages.